publish-docs.js

```diff
@@ -20,7 +20,7 @@ const dryRun = !!argv.dryRun && argv.dryRun !== 'false';
 
```

Comment:
We are copying the output to the wrong place when publishing, but I'm on it. (Run `npm run docs:publish -- --dryRun --noTeardown` to check the output.) I think I have a fix though, but will run it before pushing.

Reply:
⚡